src/main/resources/schema.sql

```diff
@@ -0,0 +1,16 @@
+create table Users(
```

Comment:
use `user_info` or `user_data` instead of `Users` (the same rules for Tickets): 1) lower case 2) simple `user` is reserved keyword in dbs

Reply:
renamed tables